cogs/assistance-cmds/faketik.3ds.md

```diff
@@ -1,6 +1,11 @@
 ---
-help-desc: Download link for faketik
+help-desc: How to download and run faketik
 aliases: faketiks
 ---
 
-3DS ticket spoofing utility, faketik: [faketik.3dsx](https://github.com/ihaveamac/faketik/releases)
+= Restore apps with missing tickets =
```

Comment:
I think it is better to add this to front matter as `title` instead, see `WritingAssistanceCommands.md` in the root of the repo

Reply:
Thanks, that was a useful document!